pyinteraph/centrality_analysis.py

```diff
@@ -106,9 +108,9 @@ def get_betweeness_cent(G, **kwargs):
 
     # Need to consider if endpoints should be used or not
     centrality_dict = nxc.betweenness_centrality(G = G,
-                                                 normalized = kwargs['norm'],
-                                                 weight = kwargs['weight_name'],
-                                                 endpoints = kwargs['endpoint'])
+                                                 normalized = kwargs['normalized'],
```

Comment:
I think we are using double-quoted strings throughout the code, maybe we should consider changing all those single quotes to double quotes.